quarkus/dist/src/main/content/bin/kc.bat

```diff
@@ -129,14 +130,33 @@ set "JAVA_RUN_OPTS=%JAVA_OPTS% -Dkc.home.dir="%DIRNAME%.." -Djboss.server.config
 
 SetLocal EnableDelayedExpansion
 
-set "ONLY_BUILD_OPTION= build"
+set "ONLY_BUILD_OPTION=build"
 set "NO_AUTO_BUILD_OPTION=optimized"
+set "NO_HELP_LONG_OPTION=--help"
```

Comment:
a bit weird naming, but works for me for now. better would be HELP_LONG_OPTION, right?

Reply:
I would leave as is.

Reply:
Changed as part of test fixes.